src/common/latest/prepare-task.ts

```diff
@@ -23,6 +23,8 @@ export const prepareTask: TaskJob = async (endpointType: EndpointType) => {
   const scanner = Scanner.getPrepareScanner(rootPath, scannerMode);
   const serverVersion = await getServerVersion(endpoint);
 
+  const extraProperties = getProperties(tl.getInput("extraProperties") ?? "");
```

Comment:
maybe this is the answer to my question above, it's natively supported now?

Reply:
yee through [this bad boy](https://www.npmjs.com/package/properties-file)